examples/cloudflare-workers/README.md

```diff
@@ -6,24 +6,39 @@ typescript worker
 
 ## How to use
 
-1. Clone and install the example
+1. Clone and install the example:
 
 ```bash
 git clone https://github.com/momentohq/client-sdk-javascript.git
 cd client-sdk-javascript/examples/cloudflare-workers
 npm install
 ```
 
-2. Create a Momento cache inside of the [console](https://console.gomomento.com)
-3. Create a [token for your cache](https://console.gomomento.com/tokens)
-4. Copy the `Rest Endpoint`, and cache name to the `wranger.toml` file
-5. Copy the `Auth Token` to the `.dev.vars` files
-5. Start the development server
+1. Create a cache inside of the [Momento console](https://console.gomomento.com/caches). Note that the Momento Cache HTTP API required for Cloudflare support is currently only available in AWS. Make note of the region you created your cache in, as you'll need it for the next step.
+1. Generate a [token for your cache](https://console.gomomento.com/tokens), making sure to choose the same AWS region you used to create the cache. You'll want to use a `Fine-Grained Access Token` with read/write permissions for the cache you created.
+   ![Console Token Screen](https://assets.website-files.com/628fadb065a50abf13a11485/64b97cb50a7e1d8d752ae539_3fU8mYh6gAhMwUYzrLOEiEXQc-KO79zANMtiH141Js2tZydZ7sFxZtr5TWLcC3OzFJTIEMZQOkLtWtBOOTEOEXmpinv1Ah3AC_LdkovI3FU7iUGY_N35cB0op1PXTNHAW0kZ-9wZ6qrCol5wrz_nuA.png)
+1. Copy the `Auth Token` and `HTTP Endpoint` values from the token generation screen for use in the next two steps.
+   ![Token generation results](https://assets.website-files.com/628fadb065a50abf13a11485/64b97cb50d9a0db6b03c40e8_JZLnsjtwN5RaGx83NX424WKmvauAuqcUD3YeWLx2LFFIwLiXHupq1XF3MOyggObfaC8LE1fQUN4b-9nDTOwGYUHugfZYqYTK92HybD2X1OsuRF-DxmJKekTWgV0SY0LzWpE9vvA0To8sGmNXkG-geQ.png)
+1. Cloudflare uses a file called wrangler.toml to configure the development and publishing of a worker. More information about Cloudflare configuration can be found [on their website](https://developers.cloudflare.com/workers/wrangler/configuration/). In the example directory, update the `wrangler.toml` file with the cache name and HTTP endpoint as they appeared in the Momento Console.
+1. Update the `.dev.vars` file in the example directory with the Momento auth token. Since this is a secret token, we don’t store it as an environment variable, instead storing it as a Cloudflare secret.
+1. Start the development server:
 
 ```bash
 npm run start
 ```
 
-6. Open your browser at [localhost:8787](http://localhost:8787)
+7. Open your browser to [localhost:8787](http://localhost:8787). The code in this example sets an item sets an item in the cache, gets it, and returns it as a JSON object.
```

Comment:
can we link & paste in the relevant lines of the source code here? like we did in the blog post? i'm worried that people will just run this without knowing what the code that gets executed in response to the http request even does. they will just get some response JSON back with no context.

also i think you have "sets an item" twice in your text.